src/gettingstarted/template/cli-install.md

```diff
@@ -0,0 +1,58 @@
+# Start with a template
+
+## Next Steps: Installing the Platform.sh CLI
+
+With all of the requirements met, install the CLI to start developing with Platform.sh.
+
+<asciinema-player src="/videos/asciinema/verify-cli-extended.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+1. **Install the CLI**
+
+    In your terminal run the following command:
+
+    * **Installing on OSX or Linux**
+
+       ```bash
+       curl -sS https://platform.sh/cli/installer | php
+       ```
+
+    * **Installing on Windows**
+
+       ```bash
+       curl https://platform.sh/cli/installer -o cli-installer.php
+       php cli-installer.php
+       ```
+
+2. **Authenticate and Verify**
+
+   Once the installation has completed, you can run the CLI in your terminal with the command
+
+   ```bash
+   platform
```

Comment:
We should show/explain what happens here.  Note that if someone hasn't setup their password yet because they used a GitHub login, they won't be able to authenticate here, will they?